integrated_channels/integrated_channel/tasks.py

```diff
@@ -89,6 +90,44 @@ def _log_batch_task_finish(task_name, channel_code, job_user_id,
         ))
 
 
+def set_enterprise_customer_site_to_request(integrated_channel):
```

Comment:
You can receive the enterprise customer object directly from the call than obtaining it inside this function. @alexjmpb